### PR TITLE
Adds *ultraflip

### PR DESCRIPTION
**Pull request by ghost**
DO THAT 1337 FLIP BRO!

it flips and spins, AT THE SAME TIME!

### CODE REVIEW

**Comment by Qustinnus** [comment]
should have a 1% chance to work 99% chance to do a fuckton of brute and make you fall.

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -492,3 +492,22 @@
 	else
 		qdel(N)
 		to_chat(user, "<span class='warning'>You don't have any free hands to make a circle with.</span>")
+
+/datum/emote/living/ultraflip
+	key = "ultraflip"
+	key_third_person = "does an awesome flip! Shouldn't that be impossible?"
```
Comment (Xhuis) [inline code/modules/mob/living/emote.dm]: This will spam the hell out of the chat boxes.

**Comment by lzimann** [comment]
This is straight copypasta from the spin emote with flip, it will not be accepted with this code(truth be told, it probably won't be accepted regardless if it's just on me).

**Comment by Anonmare** [comment]
>People aren't quick enough to input *spin and *flip at the same time


**_# LOL_**

**Comment by kevinz000** [comment]
stop.
put in the effort to spin and flip at the same time
if you want to spin and flip at the same time.

**Comment by PopNotes** [comment]
this pr is only acceptable if doing an ultraflip plays the THPS special trick sound

**Comment by MoreRobustThanYou** [comment]
@kevinz000 but it's out of sync and sometimes you do CTRL+UP twice, messing the whole thing up!

**Comment by nicbn** [comment]
Make 1/1000 chance of it gibbing you and then fine.

**Comment by lzimann** [comment]
This is just copypasta with no benefit at all.

**Pull request closed without merging**